modules/lvms-uninstalling-logical-volume-manager-operator-using-openshift-web-console.adoc

```diff
@@ -10,17 +10,17 @@ You can uninstall {lvms} using the {product-title} web console.
 
 .Prerequisites
 
-* You deleted all the applications on the clusters that are using the storage provisioned by {lvms}.
-* You deleted the persistent volume claims (PVCs) and persistent volumes (PVs) provisioned using {lvms}.
-* You deleted all volume snapshots provisioned by {lvms}.
-* You verified that no logical volume resources exist by using the `oc get logicalvolume` command.
-* You have access to the cluster using an account with `cluster-admin` permissions.
+* You have access to {product-title} as a user with `cluster-admin` permissions.
+* You have deleted the `LVMCluster` custom resource (CR).
+* You have deleted the persistent volume claims (PVCs), persistent volumes (PVs), volume snapshots, and volume clones provisioned by {lvms}.
```

Comment:
```suggestion
* You have deleted the persistent volume claims (PVCs), volume snapshots, and volume clones provisioned by {lvms}, and the applications utilizing these resources.
```
Persistent volumes (PVs) are removed by lvms once the PVCs are removed by the user.

Reply:
Updated as per the suggestion.